src/main/kotlin/org/opensearch/integrations/model/Integration.kt

```diff
@@ -0,0 +1,58 @@
+package org.opensearch.integrations.model
+
+import org.opensearch.common.io.stream.StreamInput
+import org.opensearch.common.io.stream.StreamOutput
+import org.opensearch.common.xcontent.XContentParserUtils
+import org.opensearch.core.xcontent.ToXContent
+import org.opensearch.core.xcontent.XContentBuilder
+import org.opensearch.core.xcontent.XContentParser
+import org.opensearch.observability.ObservabilityPlugin.Companion.LOG_PREFIX
+import org.opensearch.observability.util.logger
+
+data class Integration(
```

Comment:
this should be auto created from our Integration.schema file

Reply:
Out of scope for this PR, I think